### ISSUE TITLE
[RN 0.44] Warning: View.propTypes has been deprecated...

### ISSUE THREAD

**Issue by fungilation** [leecade/react-native-swiper#401]
Full console warning below. I added pointers to index.js, where it links to `react-native-swiper/src/index.js`:
```
Warning: View.propTypes has been deprecated and will be removed in a future version of ReactNative. Use ViewPropTypes instead.

reactConsoleErrorHandler @ ExceptionsManager.js:71
printWarning @ warning.js:36
warning @ warning.js:60
get @ View.js:184
(anonymous) @ index.js:103          <- http://localhost:8081/<project>/node_modules/react-native-swiper/src/index.js
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:126
_require @ require.js:110
(anonymous) @ index.js:1          <- http://localhost:8081/<project>/node_modules/react-native-swiper/src/index.js
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:126
_require @ require.js:110
(anonymous) @ components.js:28
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:126
_require @ require.js:110
(anonymous) @ index.ios.js:82
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:119
_require @ require.js:110
(anonymous) @ defaults.js:46
executeApplicationScript @ debuggerWorker.js:40
(anonymous) @ debuggerWorker.js:65
```

**Comment by leethree** [comment]
#406

**Comment by mrdezzods** [comment]
Fixed with #483
